lte/gateway/c/oai/tasks/s1ap/s1ap_state_converter.cpp

```diff
@@ -171,5 +175,75 @@ void S1apStateConverter::proto_to_s1ap_imsi_map(
       s1ap_imsi_proto.mme_ue_id_imsi_map(), s1ap_imsi_map->mme_ue_id_imsi_htbl);
 }
 
+void S1apStateConverter::supported_tai_item_to_proto(
```

Comment:
To make the code more readable and easy to debug, please put the deserialization function definition right after the serialization function.